subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/moduleconverter/dependencies/DefaultLocalConfigurationMetadataBuilder.java

```diff
@@ -85,6 +89,8 @@ public LocalConfigurationMetadata create(
         ModelContainer<?> model,
         CalculatedValueContainerFactory calculatedValueContainerFactory
     ) {
+        String name = configuration.getName();
```

Comment:
💅 Name this `configurationName` and you can use it on lines 109 and 115 instead of calling `getName()` again.